<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogEditSchool</class>
 <widget class="QDialog" name="DialogEditSchool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>92</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Изменение информации о школе</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>384</width>
     <height>92</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="nameSchoolLabel">
        <property name="text">
         <string>Название школы</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameSchoolLineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="typeSchoolComboBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Начальная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Средняя</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Старшая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Специальная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Физико-математическая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Химико-биологическая</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Гуманитарная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Музыкальная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Художественная</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Спортивная</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="streetLabel">
        <property name="text">
         <string>Улица</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="streetLineEdit"/>
      </item>
      <item>
       <widget class="QLabel" name="houseNumLabel">
        <property name="text">
         <string>Номер дома</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="houseNumLineEdit">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="applyButton">
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>Отмена</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
